localisation/000_wc_traits.csv

```diff
@@ -360,7 +360,7 @@ creature_nraqi_desc;This character is n'raqi. N'raqi, of the faceless ones, are
 creature_cthraxxi;C'thraxxi;C'thraxxi;C'thraxxi;;;;;;;;;;;x
 creature_cthraxxi_desc;This character is C'Thrax. C'Thraxxi are monstrous warbringers far larger and more resilient than other lesser faceless ones. These tremendous horrific creatures are even able to control minions of their Lords when they need it.;;Dieser Charakter ist ein C'Thrax. C'Thraxxi sind monstr�se Kriegstreiber, die weitaus gr��er und widerstandsf�higer sind als andere geringere Gesichtslose. Diese gewaltigen, schrecklichen Kreaturen sind sogar in der Lage, die Diener ihrer Herren zu kontrollieren, wenn es n�tig ist.;;;;;;;;;;;x
 subclass_dark_shaman;Dark Shaman;Sombre chaman;Dunkler Schamane;;;;;;;;;;;x
-subclass_dark_shaman_desc;This character is a dark shaman: one of those who enslaved and corrupted the elements with power of the void or Fel magic.;;Dieser Charakter ist ein dunkler Schamane: einer derjenigen, die die Elemente mit der Kraft der Leere oder Fel-Magie versklavt und verdorben haben.;;;;;;;;;;;x
+subclass_dark_shaman_desc;This character is a dark shaman: one of those who enslaved and corrupted the elements with power of the Void or Fel magic.;;Dieser Charakter ist ein dunkler Schamane: einer derjenigen, die die Elemente mit der Kraft der Leere oder Fel-Magie versklavt und verdorben haben.;;;;;;;;;;;x
 vampire;Vampire;Vampire;Vampir;;;;;;;;;;;x
 vampire_desc;This character is a vampire.;Ce personnage est un vampire.;Dieser Charakter ist ein Vampir.;;;;;;;;;;;x
```

Comment:
```suggestion
vampire_desc;Highly intelligent undead who take on bat-like features overtime. Vampires are incredibly rare, their curse little understood. They wield their power by using the blood of their victims. The greatest of the vampire groups are the San'layn and otherwise known as Darkfallen in the common tongue.;Ce personnage est un vampire.;Dieser Charakter ist ein Vampir.;;;;;;;;;;;x
```

Reply:
> The greatest of the vampire groups are the San'layn and otherwise known as Darkfallen in the common tongue.

Should be removed since it hasn't happened yet.